src/ray/object_manager/common.cc

```diff
@@ -14,19 +14,19 @@
 
 #include "ray/object_manager/common.h"
 
+#include "absl/strings/str_format.h"
+
 namespace ray {
 
-void PlasmaObjectHeader::Init() {
-#ifdef __linux__
+void PlasmaObjectHeader::Init(const ObjectID &id) {
```

Comment:
Should we pass in a unique name here instead?

Reply:
I'd rather hide the unique name logic in a function. Would it make sense to decompose it into a separate PlasmaObjectHeader method?

Reply:
Ah using the `static` counter looks good. What I meant in the original comment is that the `id` arg doesn't seem to be used (I think you were previously using it to generate the name).

Reply:
Nice catch! Done.